src/model/EpicTask.java

```diff
@@ -45,16 +52,24 @@ public void modifySubTask(SubTask subTask) { // Добавляем новую п
         if (!findSubTask) {
             this.subTasks.add(subTask);
         }
+
         this.statusTask = calcStatus();
+        calcTime();
     }
 
     private boolean isAllSubTasksByStatusTask(StatusTask statusTask) {
-        for (SubTask subTask : this.subTasks) {
-            if (!subTask.getStatusTask().equals(statusTask)) {
-                return false;
-            }
+        return subTasks.stream().allMatch(subTask -> subTask.getStatusTask().equals(statusTask));
+    }
+
+    public void calcTime() {
+        if (this.subTasks.isEmpty()) {
+            setTimeFields(NO_TIME, NO_TIME, Duration.ofMinutes(0));
+            return;
         }
-        return true;
+        LocalDateTime startTime = subTasks.stream().map(subTask -> subTask.getStartTime()).filter(time -> !time.isEqual(NO_TIME)).min(LocalDateTime::compareTo).orElse(NO_TIME);
```

Comment:
Здесь очень удобно использовать [Method References](https://www.baeldung.com/java-method-references) в методе map, стримы принято бить построчно по методам. Это все улучшает читаемость кода:
```suggestion
        LocalDateTime startTime = subTasks.stream()
            .map(Task::getStartTime)
            .filter(time -> !time.isEqual(NO_TIME))
            .min(LocalDateTime::compareTo)
            .orElse(NO_TIME);
```
Поправь также строки 70, 71